### PR TITLE
Adds a secret space lube feature

### PR DESCRIPTION
**Pull request by unid15**
~~Also moves reagent defines to /code/_reagents.dm to make them easier to find!~~ can't have nice things

### CODE REVIEW

**Comment by Shadowmech88** [comment]
I like it, but I just can't foresee any situation in which this will be seen.
Lube's toxic, too, so it's not like someone can inject themselves with a ton of it just in case they get gibbed.

**Comment by chatsonf** [comment]
Needs more gibs.
I want fat people injected with tons of lube and minttoxin to just COVER the room that they are in.

**Comment by Kurfursten** [comment]
I'm not a stickler for atomicity but I sure hope this merges **after** my PR

Otherwise :+1:

**Comment by PJB3005** [comment]
> Also moves reagent defines to /code/_reagents.dm to make them easier to find

Please we have a `__DEFINES` folder.

**Comment by ComicIronic** [comment]
Non-atomic.

**Comment by PJB3005** [comment]
http://www.nicememewebsite.website/

File: code/game/objects/effects/decals/Cleanable/humans.dm [hunk 1]
```diff
@@ -148,10 +148,10 @@ var/global/list/blood_list = list()
 
 
 
-/obj/effect/decal/cleanable/blood/gibs/proc/streak(var/list/directions)
+/obj/effect/decal/cleanable/blood/gibs/proc/streak(var/list/directions, spread_radius = 0)
 	spawn (0)
 		var/direction = pick(directions)
-		for (var/i = 0, i < pick(1, 200; 2, 150; 3, 50; 4), i++)
+		for (var/i = 0 to spread_radius)
```
Comment (ComicIronic) [inline code/game/objects/effects/decals/Cleanable/humans.dm:154]: So your gibs are now stuck to you unless you're injected with spacelube? Seems pretty crummy.

Comment (unid15) [inline code/game/objects/effects/decals/Cleanable/humans.dm:154]: Works fine last time I tested it.

Comment (ComicIronic) [inline code/game/objects/effects/decals/Cleanable/humans.dm:154]: Yeah, but the range variation is gone. All the gibs end up in a ring.

Comment (unid15) [inline code/game/objects/effects/decals/Cleanable/humans.dm:154]: There was never any range variation in the first place because of a _break_ below

Comment (ComicIronic) [inline code/game/objects/effects/decals/Cleanable/humans.dm:154]: What da fugg

**Pull request merged**